clouddriver-appengine/src/main/groovy/com/netflix/spinnaker/clouddriver/appengine/deploy/converters/UpsertAppengineLoadBalancerAtomicOperationConverter.groovy

```diff
@@ -32,6 +33,13 @@ class UpsertAppengineLoadBalancerAtomicOperationConverter extends AbstractAtomic
   }
 
   UpsertAppengineLoadBalancerDescription convertDescription(Map input) {
-    AppengineAtomicOperationConverterHelper.convertDescription(input, this, UpsertAppengineLoadBalancerDescription)
+    UpsertAppengineLoadBalancerDescription description = AppengineAtomicOperationConverterHelper
+      .convertDescription(input, this, UpsertAppengineLoadBalancerDescription) as UpsertAppengineLoadBalancerDescription
+
+    if (description.splitDescription) {
+      description.split = AppengineModelUtil.convertTrafficSplitDescriptionToTrafficSplit(description.splitDescription)
```

Comment:
Just wondering, why is this needed? Why not supply clouddriver with the right format?